.github/ISSUE_TEMPLATE/upgrade-checklist.md

```diff
@@ -0,0 +1,57 @@
+---
+name: Cosmos Hub Upgrade Checklist
+about: Create a checklist for an upgrade
+
+---
+
+## Cosmos Hub Upgrade Epic
+
+# <Upgrade Name>
+
+Check mark each item if it has been completed and ready for the upgrade. If not added yet, create an issue for each item and mark complete once it has been done and integrated.
+
+- [ ]  Upgrade to SDK version <SDK VERSION>
+- [ ]  Upgrade to IBC version <IBC VERSION>
+- [ ]  Integrate new modules
```

Comment:
Maybe link to https://github.com/cosmos/hub-eng/blob/main/module_qa/module_checklist.md or maybe we combine that with the existing module checklist template

Reply:
Cool, added a link. Wdym with adding a space to include an owner?

Reply:
@glnro lemme know, otherwise happy to merge if you are.

Reply:
lgtm